app/controllers/katello/api/v2/gpg_keys_controller.rb

```diff
@@ -71,10 +71,20 @@ def destroy
       respond_for_destroy
     end
 
+    api :GET, "/gpg_keys/:id/content", N_("Return the content of a gpg key, used directly by yum")
+    param :id, :number, :required => true
+    def content
+      if @gpg_key.content.present?
+        render(:plain => @gpg_key.content, :layout => false)
+      else
+        head(404)
```

Comment:
I don't think this else is necessary as the find_gpg_keys methods would throw a NotFound which should get translated to a 404 (although if this isn't happening, let me know)